src/test_l1_deepSleepMgr.c

```diff
@@ -647,24 +669,30 @@ void test_l1_deepSleepMgr_negative_PLAT_DS_GetLastWakeupKeyCode (void)
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
 
     // Variation 01: Call PLAT_DS_GetLastWakeupKeyCode() before initialization and check the result
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     result = PLAT_DS_GetLastWakeupKeyCode(&wakeupKeyCode);
     UT_ASSERT_EQUAL_FATAL(result, DEEPSLEEPMGR_NOT_INITIALIZED);
+    #endif
 
     // Variation 02: Call PLAT_DS_INIT() and check the result
     result = PLAT_DS_INIT();
     UT_ASSERT_EQUAL_FATAL(result, DEEPSLEEPMGR_SUCCESS);
 
     // Variation 03: Call PLAT_DS_GetLastWakeupKeyCode() with NULL pointer and check the result
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     result = PLAT_DS_GetLastWakeupKeyCode(NULL);
     UT_ASSERT_EQUAL_FATAL(result, DEEPSLEEPMGR_INVALID_ARGUMENT);
```

Comment:
This case doesnt come under enhanced error code its wrong error code. Because the scenario is handled but returning wrong value.